### DISCUSSION TITLE
Find file key doesn't work in dashboard created manually

### DISCUSSION THREAD

**Discussion by linsui** [LazyVim/LazyVim#2824] [Q&A]
I added an autocmd to show dashboard in a new tab:

```lua
vim.api.nvim_create_autocmd("TabNewEntered", {
  callback = function()
    require('dashboard'):instance()
  end,
})
```

Recently it stops working when I press `f` in the opened dashboard. It shows an error:

```
E5108: Error executing lua: ...l/share/nvim/lazy/LazyVim/lua/lazyvim/util/telescope.lua:21: attempt to index upvalue 'params' (a nil value)
stack traceback:
	...l/share/nvim/lazy/LazyVim/lua/lazyvim/util/telescope.lua:21: in function 'dump'
	...re/nvim/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:26: in function <...re/nvim/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:20>
```

How can I fix it? Thanks!

**Comment by dpetka2001** [comment]
I managed to solve this by adding in my personal configuration the following spec for `dashboard-nvim`
```lua
  {
    "nvimdev/dashboard-nvim",
    opts = function(_, opts)
      opts.config.center[1] =
        { action = [[lua LazyVim.telescope("files")()]], desc = " Find file", icon = " ", key = "f" }

      for _, button in ipairs(opts.config.center) do
        button.desc = button.desc .. string.rep(" ", 43 - #button.desc)
        button.key_format = "  %s"
      end
    end,
  },
```
I got the idea from seeing how the `config` files is being invoked in `dashboard-nvim`. Unfortunately, I just got lucky playing around with the configuration based on a hunch that we maybe need to call the function.

Pinging @folke as I'm not sure why this happens on a manually created Dashboard and don't believe that this is worthy of a PR without me understanding what's actually going on. Hopefully he can shed some light on why this happens.
